JitsiMeetJS.ts

```diff
@@ -531,24 +522,6 @@ export default {
         NetworkInfo.updateNetworkInfo({ isOnline });
     },
 
-    /**
-     * Set the contentHint on the transmitted stream track to indicate
-     * charaterstics in the video stream, which informs PeerConnection
-     * on how to encode the track (to prefer motion or individual frame detail)
-     * @param {MediaStreamTrack} track - the track that is transmitted
-     * @param {String} hint - contentHint value that needs to be set on the track
-     */
-    setVideoTrackContentHints(track, hint) {
```

Comment:
Doesn't this break the public API?

Reply:
Actually this is not documented as a public API, this should have been a private function when it was added.